.github/workflows/coverage.yaml

```diff
@@ -99,7 +99,6 @@ jobs:
           repository: ethereum/tests
           path: testpath
           sparse-checkout: |
-            GeneralStateTests
```

Comment:
Aren't these required by the coverage script so that the filled fixtures can be ran against evmone? To get the coverage diff? What did i miss?

Reply:
All GeneralStateTests are managed by static tests folder.

Reply:
Surely this change in isolation must break coverage? As the fillers disappear, but you don't generate them in the scope of this PR? What am I missing?

Reply:
Ah you take the filler from legacyTests, ok!
https://github.com/ethereum/execution-spec-tests/blob/3a6e3d7b7942fc560f3284994bd0cfb7e7aab4ac/.github/workflows/coverage.yaml#L106-L114